migrations/sqls/20221012055311-sonar-up.sql

```diff
@@ -0,0 +1,28 @@
+
+CREATE SCHEMA sonar;
+
+CREATE TABLE sonar.project (
+  id VARCHAR(255) NOT NULL,
+  quality_gate_details VARCHAR(255) NOT NULL,
+  bugs INT NULL,
+  code_smells INT NULL,
+  cognitive_complexity INT NULL,
```

Comment:
Do these need to be nullable?

Reply:
unfortunately, there was a real range on what is analysed project to project